src/AuthLDAP.php

```diff
@@ -3793,38 +3807,36 @@ public static function getDefault()
         return count($it) ? $it->current()['id'] : 0;
     }
 
+    /**
+     * @return void
+     */
```

Comment:
Please do not non-specific PHPDoc on methods that overrides a parent class method.

```suggestion
```